akka-docs-dev/rst/scala/stream-customize.rst

```diff
@@ -149,14 +149,175 @@ Using DetachedStage
 Custom graph processing junctions
 =================================
 
+To extend available fan-in and fan-out structures (graph stages) Akka Streams include :class:`FlexiMerge` and
+:class:`FlexiRoute` which provide an intuitive DSL which allows to describe which upstream or downstream stream
+elements should be pulled from or emitted to.
+
 Using FlexiMerge
 ----------------
+:class:`FlexiMerge` provides an intuitive DSL which can be used to describe from which upstream stage the merge should consume
```

Comment:
almost verbatim repetition of the sentence fragment "provides an intuitive DSL which can be used to describe from which upstream" compared to introduction paragraph.